packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -1,7 +1,10 @@
 import React, { useState, useContext } from 'react';
 import { Localized } from '@fluent/react';
 import { getLocalizedCurrency, formatPlanPricing } from '../../lib/formats';
-import { metadataFromPlan, productDetailsFromPlan } from '../../store/utils';
+import {
+  metadataFromPlan,
+  productDetailsFromPlan,
+} from 'fxa-shared/subscriptions/metadata';
```

Comment:
Lots of small changes like this throughout fxa-payments-server, since what was `store/utils` now lives in `fxa-shared/subscriptions/metadata`